src/main/resources/db/migration/V25__CREATE_AWARD.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE award
+(
+    id         SERIAL       NOT NULL PRIMARY KEY,
+    award_name VARCHAR(255) NOT NULL,
+    category   VARCHAR(255),
+    year       INTEGER          NOT NULL
+);
+
+INSERT INTO award (award_name, category, year) VALUES ('Women''s Prize for Fiction', NULL, 2020);
```

Comment:
Does 2020 represent the year a particular book won this award? If so, I don't think we should store that in this table

Reply:
Yes, I had initially set this as the year the award was established. Do you want to switch it back?

Reply:
Could we have it so that we do store the year a book won a particular award (not in this table) and not have the year the award was established? Thanks